### PR TITLE
[CSS/Instrument_list] Install FontAwesome!

### PR DESCRIPTION
**Pull request by zaliqarosli**
### Brief summary of changes
Motivated by UI issues with the instrument panel from https://github.com/aces/Loris/pull/4007, I have installed Font Awesome as a dependency on LORIS and changed all icons on the instrument panel to fa icons.

### To test this change...

- [ ] Quickly check on minor branch what it currently looks like (instrument_list)
- [ ] Checkout this branch, run `npm install`
- [ ] Got to `instrument_list`, check that icons look cute, and the lists are rendered better
- [ ] Ensure that no functionality has changed

Before & after:
screenshot 2018-11-05 12 25 57screenshot 2018-11-05 12 25 27

### CODE REVIEW

**Comment by davidblader** [comment]
do we need an extra dependency for these icons? why not just use bootstrap glyphicons?

**Comment by zaliqarosli** [comment]
@davidblader because we're trying to move away from bootstrap as being discussed in the UX SIG

File: htdocs/fontawesome [hunk 1]
```diff
@@ -0,0 +1 @@
+./../node_modules/@fortawesome/fontawesome-free/
```
Comment (johnsaigle) [inline htdocs/fontawesome]:
```suggestion
../node_modules/@fortawesome/fontawesome-free/
```
Do you need the first `./`? Seems redundant

Comment (zaliqarosli) [reply htdocs/fontawesome]: no, probably not

Comment (zaliqarosli) [reply htdocs/fontawesome]: @johnsaigle turns out we do need it

**Comment by driusan** [comment]
Merging as discussed during today's LORIS meeting.

**Pull request merged**